building-srpm-in-copr/README.md

```diff
@@ -90,3 +85,27 @@ copr_client.build_proxy.create_from_custom(ownername="lbarczio", projectname="og
                            script_builddeps=["git", "packit", "python3-wheel", "python3-pip", "python3-setuptools",
                             "python3-setuptools_scm", "python3-setuptools_scm_git_archive"])
```

Comment:
I'm wondering if Copr can actually read the BRs in the spec and install them without us passing this list

Reply:
I think we are actually talking about requirements to build _source RPMs_ not _binary RPMs_.

Reply:
> I think we are actually talking about requirements to build source RPMs not binary RPMs.

yes, I can make it more clear in the document

Reply:
oh yeah, that's a good point, we certainly don't need *all* BRs to create an SRPM although we'll definitely need some: and we already have a list: https://github.com/packit/sandcastle/blob/main/files/install-rpm-packages.yaml#L5